packages/website/src/components/SidebarItems.tsx

```diff
@@ -81,42 +65,54 @@ function resolveIcon(item: keyof GroupedMembers) {
 	}
 }
 
-export function ListSidebar({ members, selectedMember }: ListSidebarSectionProps) {
+const useStyles = createStyles((theme) => ({
+	link: {
+		fontWeight: 500,
+		display: 'block',
+		padding: 5,
+		paddingLeft: 31,
+		marginLeft: 25,
+		fontSize: theme.fontSizes.sm,
+		color: theme.colorScheme === 'dark' ? theme.colors.dark![0] : theme.colors.gray![7],
+		borderLeft: `1px solid ${theme.colorScheme === 'dark' ? theme.colors.dark![4] : theme.colors.gray![3]}`,
+
+		'&:hover': {
+			backgroundColor: theme.colorScheme === 'dark' ? theme.colors.dark![7] : theme.colors.gray![0],
+			color: theme.colorScheme === 'dark' ? theme.white : theme.black,
+		},
+	},
+}));
+
+export function SidebarItems({
+	members,
+	setOpened,
+}: {
+	members: Members;
+	setOpened: Dispatch<SetStateAction<boolean>>;
+}) {
+	const { classes } = useStyles();
 	const groupItems = groupMembers(members);
 
 	return (
 		<>
 			{(Object.keys(groupItems) as (keyof GroupedMembers)[])
 				.filter((group) => groupItems[group].length)
-				.map((group, i) => (
-					<Section iconElement={resolveIcon(group)} key={i} title={group} showSeparator={false}>
-						<div className="space-y-2">
-							{groupItems[group].map((member, i) => (
-								<div
-									key={i}
-									className="flex gap-2 whitespace-pre-wrap no-underline break-all justify-between text-blue-500 dark:text-blue-300"
-								>
-									<Link href={member.path}>
-										<a
-											className={`no-underline m-0 text-sm font-semibold ${
-												selectedMember?.containerKey === member.containerKey
-													? 'text-blue-500 dark:text-blue-300'
-													: 'text-gray-500 dark:text-gray-300 hover:text-dark-100 dark:hover:text-white'
-											}`}
-										>
-											{member.name}
-										</a>
-									</Link>
-									<div>
+				.map((group, idx) => (
+					<Section key={idx} title={group} icon={resolveIcon(group)}>
+						{groupItems[group].map((member, i) => (
+							<Link key={i} href={member.path} passHref>
+								<UnstyledButton className={classes.link} component="a" onClick={() => setOpened((o) => !o)}>
```

Comment:
different color for the selected member is removed. Is it intentional?

Reply:
For now, yes.